examples/guides/detailed-example.ipynb

```diff
@@ -377,7 +366,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.4"
+   "version": "3.8.3"
```

Comment:
Which version does this refer to?

Reply:
Yeah is 3.8, SDK supports and is regularly tested with 3.8 for some time though. Think we should also move soon to 3.8 in blocks, put it on agenda of next guild meeting.

Reply:
No need to put it on the agenda. Next meeting will be Tech debt backlog grooming anyway and there already is a ticket for that.